### ISSUE TITLE
Suggestion: develop mobile versions for tablets on IOS and Android for more $$

### ISSUE THREAD

**Issue by GreatGranPa** [aardappel/treesheets#77]
Treesheets would be IMHO a great app far tablet devices!

**Comment by aardappel** [comment]
Yup it would. Sadly TreeSheets is based on wxWidgets, which does not work on iOS/Android, or at least would require a complete UI rewrite.

Then there's the matter of storage, while desktop apps are ok with local files, a mobile app would need some form of cloud storage to go along with it.

All in all a significant effort for an app which doesn't make any money, meaning it is not likely to happen any time soon.
